include/ze_info/client.hpp

```diff
@@ -215,10 +225,31 @@ class client
         return total_gpu_time_vec;
     }
 
+    high_resolution_clock::time_point find_end_time(std::vector<start_end_time> start_end_time_vec) {
+        high_resolution_clock::time_point end_time = start_end_time_vec[0].start_time;
+        for (int i = 0; i < start_end_time_vec.size(); i++) {
+            if (end_time < start_end_time_vec[i].end_time) {
+                end_time = start_end_time_vec[i].end_time;
+            }
+        }
+        return end_time;
+    }
+
+    void save_result(double total_time) {
+        std::string filename = "./results.csv";
+        std::ofstream outfile;
+        outfile.open(filename, std::ofstream::out | std::ofstream::app);
+        outfile << "liczba threadow/kernel" << "," << "pamiec" << "," << "czas kerneli w ponkach" << "," << zenon_pool_size << "," << queries << "," << qps << "," << total_time << "\n";
```

Comment:
does this mechanism append results to the end of file or its just one time creation and save,
i think that saving results to .csv should be part of our python script